internal/cpuarm64/cpuarm64_otherwise.go

```diff
@@ -1,4 +1,4 @@
-//go:build !android || !arm64
+//go:build arm64 && !android && !darwin
```

Comment:
I'm unsure about the original intention, but I think this is the right thing to do here...

Reply:
I think we should discuss this in our 1:1 today. It may also be that a better pattern would be to have files for each combination of GOOS and GOARCH we support rather than doing smart things with conditionals (as you mentioned, we may already had some issues with conditionals, hence it seems it's better to be a little more humble here and avoid too many conditionals).

Reply:
We discussed this topic in our 1:1 and @ainghazal is going to write a small internal document to evaluate our options here. We'll explore possibilities such as vendoring `internal/cpu`.